test/vaadin-radio-group.html

```diff
@@ -380,6 +380,31 @@
           done();
         }, 100);
       });
+
+      it('should set focused attribute on focusin event dispatched', () => {
+        vaadinRadioButtonList[0].dispatchEvent(new CustomEvent('focusin', {composed: true, bubbles: true}));
+        expect(vaadinRadioButtonGroup.hasAttribute('focused')).to.be.true;
+      });
+
+      it('should set focused attribute on radio-button focus', () => {
+        vaadinRadioButtonList[0].focus();
+        expect(vaadinRadioButtonGroup.hasAttribute('focused')).to.be.true;
+      });
+
+      it('should not set focused attribute on focusin event dispatched when disabled', () => {
+        vaadinRadioButtonGroup.disabled = true;
+        vaadinRadioButtonList[0].dispatchEvent(new CustomEvent('focusin', {composed: true, bubbles: true}));
+        expect(vaadinRadioButtonGroup.hasAttribute('focused')).to.be.false;
+      });
+
+      it('should remove focused attribute on radio-button blur', () => {
+        if (!/Trident/.test(navigator.userAgent)) { // Skip IE11, fails in sauce labs
```

Comment:
Could this be solved by using `requestAnimationFrame`? Compare to how `open-wc` implements this:

https://github.com/open-wc/open-wc/blob/21b4f04e0d840f39adfba95841129ef1adc31ad7/packages/testing-helpers/helpers.js#L60-L70

Or, if that doesn't help either, let's just go ahead and wrap whole test in the `if`  statement.

Reply:
Renamed `_addActiveListeners` to `_addListeners` and wrapped the whole test in `if` statement as `requestAnimationFrame` did not work.